components/common/Header.js

```diff
@@ -232,15 +232,23 @@ class Header extends Component {
               className="d-sm-none position-fixed top-0 left-0 right-0 overflow-hidden"
```

Comment:
remove unnecessary top margin
```suggestion
              className="d-sm-none position-fixed left-0 right-0 overflow-hidden"
```